game/src/sandbox/dashboards/misc.rs

```diff
@@ -67,12 +67,7 @@ impl State<App> for ActiveTraffic {
         }
     }
 
-    fn draw_baselayer(&self) -> DrawBaselayer {
-        DrawBaselayer::Custom
-    }
-
-    fn draw(&self, g: &mut GfxCtx, app: &App) {
-        g.clear(app.cs.dialog_bg);
```

Comment:
Oh, I also got rid of the green backgrounds, because it felt more cohesive. Is there a reason we should keep it?

Relatedly, I don't really understand what `draw_baselayer` is about, but once I removed the `g.clear` here, I observed some glitchy rendering until I also removed `DrawBaselayer::Custom`.

Reply:
> I also got rid of the green backgrounds, because it felt more cohesive

Agreed, still seeing the map underneath is nicer. The "fullscreen" design with the green background was just meant to draw focus away from the map and towards the data, I think, but the new design is just as easy to read, and more cohesive as you say.

> I don't really understand what draw_baselayer is about

widgetry manages a stack of states. Most of the time, the current state is totally responsible for drawing everything, including initially clearing the screen of whatever was previously drawn there. Sometimes, we want to ask the previous state to draw, then draw on top of that. Most of the time, we want to do something default (`SharedAppState::draw_default`) -- in this case, draw the map. Occasionally though, we want to avoid the (negligible) perf hit of rendering the map at all, so we use `Custom`, which means we're responsible for doing `g.clear` too.

So what you've done is go back to `DefaultDraw`, which just renders the map, then draws on top of it. Totally fine!